PhotoPicker/src/main/java/me/iwf/photopicker/adapter/PhotoGridAdapter.java

```diff
@@ -66,6 +67,12 @@ public PhotoGridAdapter(Context mContext, List<PhotoDirectory> photoDirectories)
 
 
   @Override public void onBindViewHolder(final PhotoViewHolder holder, final int position) {
+
+
+    if (mContext instanceof Activity && ((Activity) mContext).isDestroyed()) {
```

Comment:
Activity.isDestroyed() is an API17 method.
http://developer.android.com/reference/android/app/Activity.html#isDestroyed()

I do not know which API level you want to support, but I was hoping at least API14 (Android 4.0)